README.md

```diff
@@ -7,147 +7,8 @@ NGINX Kubernetes Gateway is an open-source project that provides an implementati
 > Warning: This project is actively in development (pre-alpha feature state) and should not be deployed in a production environment.
```

Comment:
Do we need to reword this to be consistent with @brianehlert messaging?

Reply:
this is a valid concern. But I would postpone it for a future PR that will target README improvements.